src/elements/embedded-video/src/index.tsx

```diff
@@ -38,9 +54,10 @@ const EmbeddedVideo: React.FC<Props> = ({
           width: '100%',
           height: '100%'
         }}
-        src={source}
+        src={autoPlay ? constructUrl(source) : source}
```

Comment:
I think we could adjust constructUrl to take source and autoplay
```suggestion
        src={constructUrl(source, autoplay)}
```